lighthouse-core/test/fraggle-rock/gather/mock-driver.js

```diff
@@ -40,7 +40,7 @@ function createMockSession() {
 }
 
 /**
- * @param {LH.Gatherer.FRGathererInstance<LH.Gatherer.DependencyKey>['meta']} meta
+ * @param {LH.Gatherer.FRGathererInstance<keyof LH.GathererArtifacts>['meta']|LH.Gatherer.FRGathererInstance<LH.Gatherer.DefaultDependenciesKey>['meta']} meta
```

Comment:
Could we use `AnyFRGathererInstance['meta']` here?

Reply:
ah, yeah, this was the first one I changed before `AnyFRGathererInstance` existed 👍 
```suggestion
 * @param {LH.Gatherer.AnyFRGathererInstance['meta']} meta
```